include/onnxruntime/core/graph/graph.h

```diff
@@ -1141,6 +1159,35 @@ class Graph {
   */
   Status InlineFunction(Node& node);
 
+  /**
+  Directly insert the nodes in the function proto provided into the dest graph.
+  The function converts Constant nodes into the initializers in the destination graph.
+  It then creates a node in the destination graph for each of the function nodes.
+  The function names are expected to be specialized, and, therefore unique.
+
+  The Graph needs to be Resolve()d after this call.
+  @param func_to_inline
+  @param model_path model path of the graph where func_to_inline is from
+  @param dest_graph graph where we want node copies to appear
+  @returns Status indicating success or providing an error message.
+  */
+
+  Status FunctionToGraph(const ONNX_NAMESPACE::FunctionProto& func_to_inline, const Path& model_path, Graph& dest_graph);
```

Comment:
My VS studio makes use of .clangformat settings that we have checked in. I also run lintrunner before CI, so I am not sure how this is the fault of my settings.

Reply:
The clang-format settings do not enforce the line length, because if they do you lose all control over where line breaks are as it tries to make every line as long as possible with no consideration for the existing formatting. 

So either you need to manually keep things under 120 to make the linting happy, or we can turn on the clang-format line length enforcement and make the code egregiously unreadable.